### ISSUE TITLE
Does it work on non rgb bands? 

### ISSUE THREAD

**Issue by purijs** [remicres/sr4rs#2]

**Comment by remicres** [comment]
yes

**Comment by candelas762** [comment]
How do you do that? By feeding a Sentinel-2 image composed of the other bands? Do they have to be resampled to 10 m or can they be in a 20 m resolution?

**Comment by remicres** [comment]
What I mean is that the approach is sensor agnostic, although we provide a pre-trained model for RBG-NIR 10m --> 2.5m.

Train you own network, with the right input and target images. If the target is non-rgb, just disable vgg loss.
For instance, you could train a model doing SAR VV/VH @ 10m --> Near infrared @ 2.5m: you "just" have to build the right training dataset. Note that this is a particularly twisted case, just for example purpose!
